### DISCUSSION TITLE
Request thread

### DISCUSSION THREAD

**Discussion by MinekPo1** [Its-Jakey/SCPP#7] [Ideas]
I don't want to create a lot of discussions asking for new features (and I feel bad for doing it >-<), so I'm just going to dump them here.

**Comment by MinekPo1** [comment]
# Variable pointer manipulation

Proposed vm instructions:

`getVarAddress` `[var]`: sets the `a` register to the address of the variable

`setVarAddress` `[var]`: sets the address of the variable to the `a` register

This should be quite easy to implement, since doing the first thing is essentially required and the second is the reverse of the first.

**Reply by Its-Jakey** [reply]
Been at summer camp for the past week, I'm now back to working on the language

**Reply by Its-Jakey** [reply]
Done!
